egs/mgb2/asr/local/run_cleanup_segmentation.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
We do not have to include this.
Please remove it.

Reply:
done